terrawrap/utils/config.py

```diff
@@ -86,6 +89,141 @@ def parse_wrapper_configs(wrapper_config_files: List[str]) -> WrapperConfig:
     return wrapper_config_obj
 
 
+def is_config(directory: str) -> bool:
+    """
+    Checks if a wrapper file directory is a config_directory
+    :param directory: A wrapper files directory
+    :return: A boolean True if the config section of the wrapper file is not False or doesn't exist.
+    """
+    wrapper_file = os.path.join(directory, ".tf_wrapper")
+    with open(wrapper_file) as wrapper_file:
+        wrapper_config = yaml.safe_load(wrapper_file)
+        try:
+            # print("checking config")
+            # print(directory)
+            # print(wrapper_config.get("config"))
+            return wrapper_config.get("config") is not False
+        except AttributeError:
+            return True
+
+
+def has_depends_on(directory: str) -> bool:
+    """
+    Checks if a config directory has a depends_on in its tf_wrapper.
+    :param directory: The directory to check
+    :return: A boolean True if the given directory has depends_on
+    """
+    wrapper_file = os.path.join(directory, ".tf_wrapper")
+    with open(wrapper_file) as wrapper_file:
+        wrapper_config = yaml.safe_load(wrapper_file)
+        try:
+            # print("checking config")
+            # print(directory)
+            # print(wrapper_config.get("config"))
+            return wrapper_config.get("depends_on") is not None
+        except AttributeError:
+            return False
+
+
+def parse_dependencies(directory: str) -> List[str]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either all return statements in a function should return an expression, or none of them should.](https://app.codacy.com/manual/amplify-education/terrawrap/pullRequest?prid=4739316)